sdk/communication/communication-sms/package.json

```diff
@@ -130,6 +130,11 @@
     ],
     "requiredResources": {
       "Azure Communication Services account": "https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource"
-    }
+    },
+    "skip": [
```

Comment:
Why do we skip these now?

Reply:
huh, i didn't actually add the skip... i think something must've gone wrong with a merge